python/ray/_raylet.pyx

```diff
@@ -964,12 +953,14 @@ cdef class CoreWorker:
             c_vector[CTaskArg] args_vector
             c_vector[CObjectID] return_ids
 
+        # TODO(fyrestone): is_cross_language is for object serialization
```

Comment:
what's the action item for this TODO?

Reply:
This PR does not contain cross language serialization, `is_cross_language` is prepared for the future feature.

Reply:
Seems `is_cross_language` is not used yet? Maybe we can remove it right now and add it back later.

Reply:
The arg is_cross_language is not used for now. It has been removed.